sbc_compassion/models/sponsorship_correspondence.py

```diff
@@ -89,12 +91,16 @@ class SponsorshipCorrespondence(models.Model):
     destination_language_id = fields.Many2one(
         'res.lang.compassion', compute='_set_destination_language',
         inverse='_change_language', store=True)
-    original_text = fields.Text()
-    translated_text = fields.Text()
+    original_text = fields.Text(compute='_compute_page')
```

Comment:
You can add an inverse method (`inverse='_set_page'`) so that we can create a page by typing the text in the view.